src/Plugin/resource/Field/ResourceFieldEntity.php

```diff
@@ -140,6 +145,183 @@ class_implements($class_name)
   /**
    * {@inheritdoc}
    */
+  public function value(DataSourceInterface $source) {
+    // Return if this field is a callback.
```

Comment:
I'm just waterproofing our code. Imagine some newbie that has one of the keys on the plugin that decide it to be `ResourceFieldEntity` (like `sub_wrapper`), but at the same time they have a `callback`. The ideal outcome is that we return the callback.

```php
/**
  * @Resource
  * "callback" = "getStuff",
  * "sub_property" = "this_is_a_leftover",
  * …
  */
```